tests/user_commands/tests.py

```diff
@@ -194,6 +194,24 @@ def test_call_command_unrecognized_option(self):
         with self.assertRaisesMessage(TypeError, msg):
             management.call_command('dance', unrecognized=1, unrecognized2=1)
 
+    def test_call_command_with_required_parameters(self):
+        """ Verify that all variations of required option passing produce the same result"""
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Verifies that" since all tests verify things. Tests for existing correct behavior should be added in a separate commit. I think it would be better to simply test the output rather than to test that the cases are equivalent since the results could be equivalent but incorrect.